include/k4a/k4a.hpp

```diff
@@ -870,10 +901,42 @@ class transformation
         }
     }
 
+    /** Transforms depth map and a custom image into the geometry of the color camera.
+     * Throws error on failure
+     *
+     * \sa k4a_transformation_depth_image_to_color_camera_custom
+     * Creates a new image with the output.
+     */
+    std::pair<image, image>
+    depth_image_to_color_camera_custom(const image &depth_image,
+                                       const image &custom_image,
+                                       k4a_transformation_interpolation_type_t interpolation_type,
+                                       uint32_t invalid_custom_value) const
+    {
+        image transformed_depth_image = image::create(K4A_IMAGE_FORMAT_DEPTH16,
+                                                      m_color_resolution.width,
+                                                      m_color_resolution.height,
+                                                      m_color_resolution.width *
+                                                          static_cast<int32_t>(sizeof(uint16_t)));
+        image transformed_custom_image = image::create(K4A_IMAGE_FORMAT_CUSTOM,
```

Comment:
>K4A_IMAGE_FORMAT_CUSTOM [](start = 55, length = 23)

this should be the custom_image's format, you can use the image_get_format to retrieve it from the original custom image

Reply:
Thank you for the advice.